graphics/rawtherapee/Portfile

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           active_variants 1.1
+PortGroup           app 1.0
+
+name                rawtherapee
+app.name            RawTherapee
+version             5.8
+categories          graphics
+platforms           darwin
+license             GPL-3
+maintainers         @jasonliu-- openmaintainer
+
+homepage            http://rawtherapee.com
+description         raw photo processing program
+long_description    ${app.name} is a powerful, cross-platform \
+                    ${description}. It is designed for developing raw \
+                    files from a broad range of digital cameras, as \
+                    well as HDR DNG files and non-raw image formats \
+                    (JPEG, TIFF, and PNG).
+
+master_sites        http://rawtherapee.com/shared/source/
+use_xz              yes
+checksums           rmd160  79a01872a7edcc64e78b3d05d17d0a38fcb4f83f \
+                    sha256  360528a0aae922eb5af8742f415475fb91b6d62a739da5f2804828f04ec40853 \
+                    size    12653148
+
+compiler.cxx_standard 2011
+
+# We need to blacklist Apple Clang, because none of the compilers
+# provided by Xcode have any support for OpenMP.
+# (See https://trac.macports.org/wiki/CompilerSelection)
+# I did try using Apple Clang with OpenMP through the use of the
+# '-Xpreprocessor -fopenmp' trick (see
+# https://lists.macports.org/pipermail/macports-dev/2021-November/043902.html),
+# but the compile still fails (with an "unable to execute command:
+# Segmentation fault: 11" error), at least when tested on macOS 10.11.
+compiler.blacklist-append clang
```

Comment:
I think instead of all of this, you should be able to set `compiler.openmp_version 4.5` (of course specifying the correct minimum version). Then you don't need to add the `libomp` dependency anymore. The code for this is in [base](https://github.com/macports/macports-base/blob/master/src/port1.0/portconfigure.tcl).

Reply:
I didn't realize that `compiler.openmp_version` also blacklisted Apple Clang. I've changed both actions to a single line:

https://github.com/macports/macports-ports/blob/4f01bf60e470b85ebe53ef1ba062dc6369ce101e/graphics/rawtherapee/Portfile#L32